src/libcollections/string.rs

```diff
@@ -809,14 +986,23 @@ impl String {
         self.len() == 0
     }
 
-    /// Truncates the string, returning it to 0 length.
+    /// Truncates this `String`, removing all contents.
+    ///
+    /// While this means the `String` will have a length of zero, it does not
+    /// touch its capacity.
     ///
     /// # Examples
     ///
+    /// Basic usage:
+    ///
     /// ```
-    /// let mut s = "foo".to_string();
+    /// let mut s = String::from("foo");
```

Comment:
Sure. Since I'm going through a sweep of std in whole, I've been trying to just do the work individually, as it's easier. Same goes for the assert order issue.